sonic-net/sonic-buildimage
# [Nokia][devicedata] Modified the port autoneg default setting for Nokia-7215 platform

#### Why I did it
autoneg is not supported in the previous release 202012 on Nokia-7215 platform.  To migrate to the 202205 image with autoneg support, we need to disable the autoneg to allow the link to be up when issue load minigraph.  This requires to change the autoneg setting to be off in the port_config.ini file.

#### How I did it
Modify the port_config.ini to set the autoneg off. 

#### How to verify it
1) Running the new image,  with load mingraph.xml, execute "show int autonet status"  should show autoneg disabled
```
admin@ixs-7215-pizza9:~$ show inter autoneg status
  Interface    Auto-Neg Mode    Speed    Oper    Admin
-----------  ---------------  -------  ------  -------
  Ethernet0         disabled       1G    down       up
  Ethernet1         disabled      1G    down       up
  Ethernet2         disabled      1G    down       up
  Ethernet3         disabled      1G    down       up
  Ethernet4         disabled       1G    down       up
  Ethernet5         disabled       1G    down       up
```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
